recipe/meta.yaml

```diff
@@ -37,10 +38,17 @@ test:
     - pip
 
 about:
-  home: http://github.com/miurahr/pyppmd
+  description: |
+    Classes and functions for compressing and decompressing text data, using
+    PPM(Prediction by partial matching) compression algorithm which has several
+    variations of implementations.
+  home: https://codeberg.org/miurahr/pyppmd
+  dev_url: https://codeberg.org/miurahr/pyppmd
+  doc_url: https://codeberg.org/miurahr/pyppmd/src/branch/main/README.rst
   summary: PPMd compression/decompression library
   license: BSD-3-Clause
```

Comment:
license should be LGPLv2
see https://github.com/miurahr/pyppmd/blob/releases/1.0/LICENSE

Reply:
I found another mistake, will close for now